test/test_gradient_moments.py

```diff
@@ -18,8 +37,19 @@ def test_EFLinearOperator_matvec(case, adjoint: bool):
 
 
 def test_EFLinearOperator_matmat(case, adjoint: bool, num_vecs: int = 3):
-    op = EFLinearOperator(*case)
-    op_functorch = functorch_empirical_fisher(*case).detach().cpu().numpy()
+    model_func, loss_func, params, data, batch_size_fn = case
+
+    op = EFLinearOperator(
+        model_func, loss_func, params, data, batch_size_fn=batch_size_fn
+    )
+    op_functorch = (
+        functorch_empirical_fisher(
+            model_func, loss_func, params, data, batch_size_fn, "x"
```

Comment:
Specify `input_key='x'` and `batch_size_fn=batch_size_fn` to improve readability.